### PR TITLE
Update foreman required version for rubygem-katello

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

**Comment by zjhuntin** [comment]
@ekohl I added the upper bound, but wanted to make sure this is what you were thinking.

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -24,6 +24,7 @@ Requires: katello-selinux
 Requires: foreman-postgresql
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
+Requires: foreman < 2.2
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec:27]: Could be good to follow the same pattern for foreman_max_version

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 2]
```diff
@@ -234,6 +235,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/webpack
 
 %changelog
+* Wed May 06 2020 Zach Huntington-Meath <zhunting@redhat.com> - 3.16.0-0.12.pre.master
+- Update foreman minimum version
```
Comment (ekohl) [inline packages/katello/rubygem-katello/rubygem-katello.spec:239]: How about
```suggestion
- Update foreman version bounds
```

File: packages/katello/rubygem-katello/rubygem-katello.spec (hunk 1 above)
Comment (ekohl) [reply packages/katello/rubygem-katello/rubygem-katello.spec:27]: That, or perhaps you can do some math so it becomes `%{foreman_min_version} + 0.1`. Not sure if RPM specs allow this.

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by evgeni** [comment]
this needs a rebase now, sorry

**Comment by ehelms** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

**Comment by zjhuntin** [comment]
[test rpm]

File: packages/plugins/ansiblerole-insights-client/insights-client-1.7.1.tar.gz [hunk 3]
```diff
@@ -0,0 +1 @@
+../../../.git/annex/objects/V8/WV/SHA256E-s11781--6c22e8ea717b86e5e77a42b2a42dfd2ab6b9beae0171bb78ff230bae6e9c322b.tar.gz/SHA256E-s11781--6c22e8ea717b86e5e77a42b2a42dfd2ab6b9beae0171bb78ff230bae6e9c322b.tar.gz
```
Comment (ehelms) [inline packages/plugins/ansiblerole-insights-client/insights-client-1.7.1.tar.gz:1]: Don't think you meant t include this file with this update

**Comment by ehelms** [comment]
Ah ha! So I don't think you need el8-katello-nightly in the lookasides, its just all around missing from here:

https://github.com/theforeman/foreman-packaging/blob/rpm/develop/repoclosure/yum.conf

**Pull request merged**